CHANGELOG.rst

```diff
@@ -4,10 +4,19 @@ Changelog
 v3.0.0.dev
 ----------
 Complete overhaul of 21cmFAST, including a robust python-wrapper and interface,
-caching mechanisms, and public repository with continuous integration.
+caching mechanisms, and public repository with continuous integration. All changes
+and equations for this version are found in https://arxiv.org/abs/2003.04442
 
```

Comment:
not all of the changes are here..  for the physics, also the velocities from Julian's paper are included.  there there are photon conservation, etc...  i would include a few papers, and remove "all"

Reply:
I'll leave those to their PRs